hashring/hashring.go

```diff
@@ -76,6 +80,7 @@ func New(hashfunc func([]byte) uint32, replicaPoints int) *HashRing {
 		hashfunc: func(str string) int {
 			return int(hashfunc([]byte(str)))
 		},
+		logger: logging.Logger("ring"),
```

Comment:
I'm pretty sure the hostname is visible in log analysis, but it is missing the port number. When we run multiple instances on 1 machine (port 0) it is hard to reason about the identity of the ringpop member that logged the line which I find sometimes very handy.